src/applications/facility-locator/components/SearchControls.jsx

```diff
@@ -68,6 +69,7 @@ const SearchControls = props => {
 
   const handleSubmit = e => {
     e.preventDefault();
+    e.stopPropagation();
```

Comment:
Added because of va-button effects, but also limited with the type of submission from the button.